### PR TITLE
Add card emoji tests in Java streams sets

### PR DESCRIPTION
**Pull request by aqsa505**
Signed-off-by: Aqsa Malik <aqsa.malik@bnymellon.com>

### CODE REVIEW

File: deck-of-cards-kata-solutions/src/test/java/bnymellon/codekatas/deckofcards/sortedset/immutable/JavaStreamsDeckOfCardsAsSortedSetTest.java [hunk 1]
```diff
@@ -111,7 +123,14 @@ public void dealHands()
         Deque<Card> jdk2Shuffled = this.jdk2Deck.shuffle(new Random(1));
         List<Set<Card>> jdk1Hands = this.jdk1Deck.dealHands(jdk1Shuffled, 5, 5);
         List<Set<Card>> jdk2Hands = this.jdk2Deck.dealHands(jdk2Shuffled, 5, 5);
-        Assertions.assertEquals(jdk1Hands, jdk2Hands);
```
Comment (donraab) [inline deck-of-cards-kata-solutions/src/test/java/bnymellon/codekatas/deckofcards/sortedset/immutable/JavaStreamsDeckOfCardsAsSortedSetTest.java:114]: This doesn't need to be removed (yet).

**Review by donraab — CHANGES_REQUESTED** [review]
Hi @aqsa505, thanks for the contribution! One minor change.

File: deck-of-cards-kata-solutions/src/test/java/bnymellon/codekatas/deckofcards/sortedset/immutable/JavaStreamsDeckOfCardsAsSortedSetTest.java (hunk 1 above)
Comment (aqsa505) [reply deck-of-cards-kata-solutions/src/test/java/bnymellon/codekatas/deckofcards/sortedset/immutable/JavaStreamsDeckOfCardsAsSortedSetTest.java:114]: added back!

**Review by donraab — APPROVED** [review]
LGTM! Thanks for the contribution @aqsa505 !

**Pull request merged**